features/waardetabel.feature

```diff
@@ -0,0 +1,248 @@
+
+Functionaliteit: Waardetabel met code en omschrijving
+    Een veld dat wordt gevuld vanuit een in de BRP opgeslagen code die refereert naar een code in een 
+    landelijke tabel, wordt geleverd met zowel de code als de bijbehorende omschrijving.
+
+    Naast velden met een waarde uit een landelijke tabel zijn er ook velden die in de BRP zijn gedefinieerd 
+    met vaste mogelijke waarden. Deze mogelijke waarden en de bijbehorende omschrijvingen staan beschreven 
+    in de tabelwaarden feature.
+
+    Mogelijke codes en omschrijvingen zijn te raadplegen met de API BRP tabellen bevragen.
+    Zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-tabellen-bevragen
+
+  @gba
+  Rule: Bij een veld gedefinieerd als Waardetabel op een landelijke tabel wordt de omschrijving geleverd die in die tabel hoort bij de code
+
+    Abstract Scenario: Omschrijving bij code voor <element>
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      | <element>           | <waarde>  |
+      En het systeem heeft landelijke tabel '<tabel>' met de volgende gegevens
+      | code     | omschrijving   |
+      | <waarde> | <omschrijving> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | <veld>                          |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde         |
+      | <veld>.code         | <waarde>       |
+      | <veld>.omschrijving | <omschrijving> |
+
+      Voorbeelden:
+      | element                              | tabel                    | veld                       | waarde | omschrijving                         |
+      | Gemeente van inschrijving (09.10)    | 33 Gemeenten             | gemeenteVanInschrijving    | 1326   | 's Heer Hendriks Kinderen            |
+      | Indicatie gezag minderjarige (32.10) | 61 Gezagsverhoudingtabel | indicatieGezagMinderjarige | 1D     | Ouder1 en een derde hebben het gezag |
+
+    Abstract Scenario: Omschrijving bij code voor <element>
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      | <element>           | <waarde>  |
+      En het systeem heeft landelijke tabel '<tabel>' met de volgende gegevens
+      | code     | omschrijving   |
+      | <waarde> | <omschrijving> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | <groep>.<veld>                  |
```

Comment:
Zoals ik het zie beschrijven het vorige scenario en dit scenario dezelfde functionaliteit t.a.v. het toepassen van waardetabellen. Volgens mij dit geen aanvulling vanuit het functionele parspectief van hoe waardetabellen moeten worden toegepast

Reply:
de scenario's worden niet alleen gebruikt voor illustratie van de functionaliteit, maar kunnen ook gebruikt worden voor tests. Met dit extra scenario worden alle waardetabellen in de resource worden getest.